pkg/crc/machine/machine.go

```diff
@@ -73,6 +74,24 @@ func getCrcBundleInfo(bundlePath string) (*bundle.CrcBundleInfo, error) {
 	return bundle.Extract(bundlePath)
 }
 
+func checkDiskImageSize(bundleinfo *bundle.CrcBundleInfo) error {
+	diskImagePath := bundleinfo.GetDiskImagePath()
+	expectedSize, err := bundleinfo.GetDiskSize()
+	if err != nil {
+		return err
+	}
+	logging.Infof("Checking size of the disk image %s ...", diskImagePath)
```

Comment:
can we try not to put all functionality in machine.go ?

it would be better if we only perform a boolean action in machine.go, but put the code in a separate file or even inside a more logical place.